tcollector.py

```diff
@@ -682,6 +689,33 @@ def send_data(self):
         # the packets out of the kernel's queue
 
 
+    def send_data_to_signalfx(self):
+        try:
+            gauges = []
+            for line in self.sendq:
+                split_line = self.add_tags_to_line(line).split()
+                metric, timestamp, value = split_line[:3]
+                tags = dict(tag.split('=') for tag in split_line[4:])
+                gauges.append({
+                    'metric': metric,
+                    'value': self.num(value),
+                    'timestamp': int(timestamp)*1000,
+                    'dimensions': tags})
+            self.sfx.send(gauges=gauges)
+        except:
+            LOG.error('Failed to send data to signalfx: %s' % sys.exc_info()[0])
```

Comment:
I didn't want to cause a runtime exception in case there's a problem with SignalFX's API, at least while we're still logging to OpenTSDB.

It might be better to wait until we move over for good (if we do) before raising this.